examples/train_csv.py

```diff
@@ -0,0 +1,84 @@
+import csv
+import time
+from datetime import datetime, timezone, timedelta
+from typing import List, Dict, Tuple, Any
+from fsrs_rs_python import FSRS, FSRSItem, FSRSReview
+
+
+def main():
+    # Read revlog.csv
+    # Please download from
+    # https://github.com/open-spaced-repetition/fsrs-rs/files/15046782/revlog.csv
+    with open("./revlog.csv", "r") as file:
+        records = list(csv.DictReader(file))
+
+    print(f"revlogs.len() = {len(records)}")
+    start_time = time.time()
+
+    # Group by card_id
+    reviews_by_card = group_reviews_by_card(records)
+
+    # Convert to FSRSItems
+    fsrs_items = [
+        item
+        for items in map(convert_to_fsrs_item, reviews_by_card.values())
+        for item in items
+    ]
+    print(f"fsrs_items.len() = {len(fsrs_items)}")
+
+    # Create FSRS instance and optimize
+    fsrs = FSRS([])
+    optimized_parameters = fsrs.compute_parameters(fsrs_items)
+    print("optimized parameters:", optimized_parameters)
+    end_time = time.time()
+    print(f"Full training time: {end_time - start_time:.2f}s\n")
+
+
+def group_reviews_by_card(records: List[Dict]) -> Dict[str, List[Tuple[datetime, int]]]:
```

Comment:
here Dict could be https://docs.python.org/zh-cn/3.13/library/typing.html#typing.TypedDict